Swift-Base/AppDelegate.swift

```diff
@@ -29,6 +29,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         #endif
         
+        #if DEBUG
+            // print documents directory and device ID
+            print("\n*******************************************\nDOCUMENTS\n\(NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true)[0])\n*******************************************\n")
```

Comment:
I think we must add functions here like "setupMagicalRecord", "setupSDWebImage" and others
And debug preparing too